tests/API/Management/UsersTest.php

```diff
@@ -4,61 +4,189 @@
 use Auth0\SDK\API\Management;
 use Auth0\Tests\API\BasicCrudTest;
 
-class UsersTest extends BasicCrudTest {
+/**
+ * Class UsersTest.
+ *
+ * @package Auth0\Tests\API\Management
+ */
+class UsersTest extends BasicCrudTest
+{
+    /**
+     * Unique identifier name for Users.
+     *
+     * @var string
+     */
+    protected $id_name = 'user_id';
 
-    protected $email;
+    /**
+     * Should the results returned by getAll be searched for the created user?
+     *
+     * @var bool
+     */
     protected $findCreatedItem = false;
```

Comment:
also, doesn't this apply for all the entities with a CREATE method? Should it be generalized like the `rand` in the super class?

Reply:
No, it's an option of sorts